vips/vip-converter/vip-converter-testnet/commands.ts

```diff
@@ -0,0 +1,153 @@
+  const config: CallPermission = {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [target, signature, caller],
+  };
+
+  return config;
+};
+
+function getIncentiveAndAccessibility(tokenIn: string, tokenOut: string): IncentiveAndAccessibility {
+  const validTokenIns = [BaseAssets[2], BaseAssets[3], BaseAssets[4], BaseAssets[5]];
+
+  // Every conversion of the baseAsset for USDT in a SingleTokenConverter with a baseAsset != USDT is enabled only for converters,
+  // because the RiskFundConverter of the USDTPrimeConverter should be able to cover those conversions
+  if (validTokenIns.includes(tokenIn) && tokenOut === BaseAssets[0]) {
+    return [0, 2]; // ONLY_FOR_CONVERTERS
+  } else {
+    return [0, 1]; // ALL
+  }
+}
+
+function generateAcceptOwnershipCommands(ConvertersArray: string[]): AcceptOwnership[] {
+  const acceptOwnershipCommandsArray: AcceptOwnership[] = [];
+
+  for (const converter of ConvertersArray) {
+    const config: AcceptOwnership = {
+      target: converter,
+      signature: "acceptOwnership()",
+      params: [],
+    };
+
+    acceptOwnershipCommandsArray.push(config);
+  }
+
+  return acceptOwnershipCommandsArray;
+}
+
+function generateConverterCommands(ConvertersArray: string[]): ConverterCommand[] {
+  const commandsArray: ConverterCommand[] = [];
+
+  for (const converter of ConvertersArray) {
+    // Add AddConverterNetwork command
+    const addConverterNetworkConfig: ConverterCommand = {
+      target: converter,
+      signature: "setConverterNetwork(address)",
+      params: [CONVERTER_NETWORK],
+    };
+
+    // Add AddTokenConverter command
+    const addTokenConverterConfig: ConverterCommand = {
+      target: CONVERTER_NETWORK,
+      signature: "addTokenConverter(address)",
+      params: [converter],
+    };
+
+    commandsArray.push(addConverterNetworkConfig);
+    commandsArray.push(addTokenConverterConfig);
+  }
+
+  return commandsArray;
+}
+
+function generateCallPermissionCommands(ConvertersArray: string[]): CallPermission[] {
+  const callPermissionCommandsArray: CallPermission[] = [];
+
+  for (const converter of ConvertersArray) {
+    for (const timelock of TimelocksArray) {
+      const config1 = grant(converter, "setConversionConfig(address,address,ConversionConfig)", timelock);
+      const config2 = grant(converter, "pauseConversion()", timelock);
+      const config3 = grant(converter, "resumeConversion()", timelock);
+      const config4 = grant(converter, "setMinAmountToConvert(uint256)", timelock);
+
+      callPermissionCommandsArray.push(config1);
+      callPermissionCommandsArray.push(config2);
+      callPermissionCommandsArray.push(config3);
+      callPermissionCommandsArray.push(config4);
+    }
+
+    callPermissionCommandsArray.push(grant(converter, "pauseConversion()", GUARDIAN));
+    callPermissionCommandsArray.push(grant(converter, "resumeConversion()", GUARDIAN));
+  }
```

Comment:
Given there are 6 converters, and the name of the functions are not generic (like `pause` or `resume`), what do you think if we grant permission using the `ZERO_ADDRESS` instead of the specific `converter` address? This way we would reduce the number of commands from 86 to 14